src/__tests__/e2e.test.js

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable react/no-multi-comp,react/prop-types,react/prefer-stateless-function  */
 import React from 'react';
-import { shallow, mount } from 'enzyme';
+import { shallow, mount } from './helpers/testSetup';
```

Comment:
I would rather keep the imports directly from `enzyme` instead of a custom setup file. I think to specify the raf polyfill you can do so via jest configuration ([via a `jest.config.js` file at the root of the repo](https://facebook.github.io/jest/docs/en/configuration.html)), that's where you can setup the raf polyfill as well:

```js
/* jest.config.js */
module.exports = {
  setupFiles: [
    "raf/polyfill",
    "./setup/enzymeAdapter.js"
  ]
}
```